src/styles/_calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 10px;
+  width: calc((#{$day-size} * #{$columns}) + (#{$gap} * #{$columns - 1}));
+  background-color: $calendar-background-color;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    border: 1px solid $border-color;
+    width: $day-size;
+    height: $day-size;
+    background-color: $day-background-color;
+    transition:
+      background-color 0.5s,
+      transform 0.5s;
+    &::before {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      margin: auto;
+    }
+    &:hover {
+      cursor: pointer;
+      background-color: $hover-background-color;
+      transform: translateY(-20px);
+      transition:
+        background-color 0.5s,
+        transform 0.5s;
```

Comment:
The transition property is being repeated in the &:hover selector. Since the transition is already defined in the .calendar__day selector, it is not necessary to redefine it in &:hover unless you want to override it with different values.